e2e/content-services/document-list/document-list-actions.e2e.ts

```diff
@@ -223,10 +223,17 @@ describe('Document List Component - Actions', () => {
             it('[C260060] Should be able to open a file/folder through double click action - folder', async () => {
```

Comment:
is it necessary to have the folder and subfolders created during the test? Can't they be created using API calls in beforeAll? Reading the title of this test case, i would say the folders are just preconditions.

Reply:
I didn;'t change the test just the way it was doing , in this way the API are faster. We can refactor the test later